srcpkgs/rocksdb/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'rocksdb'
+pkgname=rocksdb
+version=5.17.2
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release -DWITH_TESTS=0 -DUSE_RTTI=0 -DWITH_LZ4=1"
+hostmakedepends="git"
```

Comment:
What does it need git for?

Reply:
actually, it was a warning at "configure" time. i'll remove it.